src/v/cluster/topics_frontend.cc

```diff
@@ -1321,4 +1323,69 @@ ss::future<bool> topics_frontend::replica_set_exists(
     co_return true;
 }
 
+namespace {
+bool in_broker_list(
+  const model::broker_shard& bshard,
+  const std::vector<model::broker_shard>& list) {
+    return std::find(list.begin(), list.end(), bshard) != list.end();
+}
+} // namespace
+
+ss::future<partition_reassignments>
+topics_frontend::list_partition_reassignments(
+  model::ntp ntp, model::timeout_clock::time_point tout) {
+    auto result = co_await stm_linearizable_barrier(tout);
```

Comment:
Why is this necessary? I believe its ok to return the assignments as they stand currently.

Reply:
That was leftover code from another topics_frontend method. I'll remove.